web/src/components/nav/NavProfile.tsx

```diff
@@ -19,16 +19,22 @@ import {
     useSidebar,
 } from '@/components/ui/sidebar';
 import { Skeleton } from '@/components/ui/skeleton';
-import { getUserInfoMeGet, logoutAuthLogoutPost } from '@/gen';
+import { logoutAuthLogoutPost } from '@/gen';
+import { useMe } from '@/hooks/use-me';
 
 export function NavProfile() {
-    const {
-        data: user,
-        isPending: userPending,
-        isError: userError,
-    } = useQuery({
-        queryKey: ['user'],
-        queryFn: () => getUserInfoMeGet(),
+    const queryClient = useQueryClient();
+    const { data: user, isPending: userPending, isError: userError } = useMe();
+
+    const logoutMutation = useMutation({
+        mutationFn: () => logoutAuthLogoutPost(),
+        onSuccess: () => {
+            queryClient.invalidateQueries({ queryKey: ['me'] });
+            window.location.href = '/login';
```

Comment:
I know I wrote this redirect initially, but seeing it now reminded me we need to decide what 'home' should be, as that's where we should send someone who logs out or visits the site for the first time. Not sure if that should be one of /review, /decks, or /profile in guest mode, or if it should be some future landing page.

Not for this PR though.